docs/topics/assembly_developing-and-deploying-vertx-application.adoc

```diff
@@ -20,6 +20,12 @@ In addition to xref:example-rest-http-vertx[using an example], you can create ne
 //Developing an application
 include::proc_developing-runtime-application.adoc[leveloffset=+1]
 
+//Supported OS and JDK images
```

Comment:
This doesn't fit very well here IMO. If this is for OpenShift only, I think it should be in the OpenShift deployment assembly..

Reply:
Moved.